doc/sphinx/administration_portal/client/vpbx/routing_endpoints/ivrs.rst

```diff
@@ -81,8 +107,26 @@ an error and will trigger the **Error process**):
 
 .. rubric:: Entries are regular expressions
 
+*****************
+Entry information
+*****************
 You can specify IVR entries as Regular Expressions. If entry is just
 a numeric value, it will be handled as a sequence of digits, otherwise it
-will be handled a regular expression. This can be handy if you have the
+will be handled as a regular expression. This can be handy if you have the
 same behaviour for a group of dialed numbers.
 
+    Entry
+        Sequence of digits or regular expression.
+
+    Success locution
+        Locution to be played in case of success.
+
+    Display name
+        Visible name for the IVR entry
```

Comment:
_This value will be displayed in the called terminals_

(Taken from Queues section)